static/app/views/performance/newTraceDetails/traceHeader/highlights.tsx

```diff
@@ -0,0 +1,133 @@
+import styled from '@emotion/styled';
+
+import {
+  EnvironmentHighlight,
+  HighlightsIconSummary,
+  ReleaseHighlight,
+} from 'sentry/components/events/highlights/highlightsIconSummary';
+import {space} from 'sentry/styles/space';
+import type {EventTransaction} from 'sentry/types/event';
+import type {Organization} from 'sentry/types/organization';
+import type {Project} from 'sentry/types/project';
+import type {UseApiQueryResult} from 'sentry/utils/queryClient';
+import type RequestError from 'sentry/utils/requestError/requestError';
+import {useTraceItemDetails} from 'sentry/views/explore/hooks/useTraceItemDetails';
+import {
+  OurLogKnownFieldKey,
+  type OurLogsResponseItem,
+} from 'sentry/views/explore/logs/types';
+import {TraceItemDataset} from 'sentry/views/explore/types';
+import {TraceHeaderComponents} from 'sentry/views/performance/newTraceDetails/traceHeader/styles';
+import {
+  TraceShape,
+  type TraceTree,
+} from 'sentry/views/performance/newTraceDetails/traceModels/traceTree';
+
+function LogsHighlights({
+  logs,
+  organization,
+  project,
+}: {
+  logs: OurLogsResponseItem[];
+  organization: Organization;
+  project: Project | undefined;
+}) {
+  const log = logs[0];
+
+  const {data, isPending} = useTraceItemDetails({
+    traceItemId: String(log![OurLogKnownFieldKey.ID]),
+    projectId: String(log![OurLogKnownFieldKey.PROJECT_ID]),
+    traceId: String(log![OurLogKnownFieldKey.TRACE_ID]),
```

Comment:
If `logs[0]` must always exist, then you can update the type to `logs: [OurLogsResponseItem, ...OurLogsResponseItem[]];`. This allows us to avoid using the `!` not null assertion.

Reply:
Addressed in the above comment.